docs/batch_inference/index.md

```diff
@@ -0,0 +1,16 @@
+(batch_inference_overview)=
+# Batch inference
+
```

Comment:
im not clear why we need 3 files (hierarchy) for this, the `test_and_deploy_batch_model_inference` doesn't add much to batch inference. I guess we should just have a section about bringing your own model (do `project.log_model()`) vs using an mlrun model artifact (from a training job)

Reply:
I agree it doesn't add a lot of info, for now made it to a single one, we can decide if to split once we have more content